app/lib/features/chat_ng/widgets/chat_bubble.dart

```diff
@@ -105,19 +113,40 @@ class ChatBubble extends StatelessWidget {
                 ),
                 child: Column(
                   crossAxisAlignment: CrossAxisAlignment.start,
+                  mainAxisSize: MainAxisSize.min,
                   children: [
                     child,
-                    if (isEdited) ...[
-                      const SizedBox(height: 4),
-                      Align(
-                        alignment: Alignment.bottomRight,
-                        child: Text(
-                          L10n.of(context).edited,
-                          style: chatTheme.emptyChatPlaceholderTextStyle
-                              .copyWith(fontSize: 12),
-                        ),
-                      ),
-                    ],
+                    const SizedBox(height: 8),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: [
+                        if (isEdited) ...[
+                          Text(
+                            L10n.of(context).edited,
+                            style: chatTheme.emptyChatPlaceholderTextStyle
+                                .copyWith(fontSize: 12),
+                          ),
+                        ],
```

Comment:
one item in a list doesn't need `...[` notation

```suggestion
                        if (isEdited)
                          Text(
                            L10n.of(context).edited,
                            style: chatTheme.emptyChatPlaceholderTextStyle
                                .copyWith(fontSize: 12),
                          ),
```